src/init.js

```diff
@@ -0,0 +1,75 @@
+/*------------------------------------------------------------------------------------------------\
+  Initialize adverse event explorer.
+\------------------------------------------------------------------------------------------------*/
+
+export function init(data) {
+    var element = d3.select(this.element);
```

Comment:
or perhaps `canvas` (which is used elsewhere). Idea is to differentiate between the css selector (which is a string) and the d3 selection (which is a fairly complex object)